quilt/test/test_integration.py

```diff
@@ -3,17 +3,25 @@
 """
 import pytest
 
-
 from ..tools import command
 from .utils import BasicQuiltTestCase
 from .integration import skip
 
 @skip
 class IntegrationTest(BasicQuiltTestCase):
+    """only runs if --integration ENV_URL is provided to pytest"""
     def test_prod_install(self):
+        # this is done in preference to os.environ.get since there is no
+        # guarantee that command.QUILT_PKG_URL == os.environ.get['QUILT_PKG_URL']
+        # WARNING: not thread safe
+        old_env = command.QUILT_PKG_URL
+        env = pytest.config.getoption("--integration")
+        command.QUILT_PKG_URL = env
```

Comment:
You should use `with patch(...):` here. Simpler, and guarantees that the old value gets restored at the end.